lekce2/14.py

```diff
@@ -0,0 +1,10 @@
+import sys
+word = sys.argv[1]
+text = word.split('_')
+capital1 = [cap.capitalize() 
+for cap in text]
```

Comment:
Napis to na jeden radek, tohle vypada jak zacatek obycejneho cyklu mimo list comprehension.

Reply:
jj, to jsem se nejak uklepl.